extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -313,14 +313,16 @@ public static class Jwt {
             public Map<String, String> claims = new HashMap<>();
 
             /**
-             * Signature algorithm, also used for the {@link #keyFile} property.
-             * Supported values: RS256, RS384, RS512, PS256, PS384, PS512, ES256, ES384, ES512, HS256, HS384, HS512.
+             * The signature algorithm used for the {@link #keyFile} property.
+             * Supported values: `RS256` (default), `RS384`, `RS512`, `PS256`, `PS384`, `PS512`, `ES256`, `ES384`, `ES512`,
+             * `HS256`,
+             * `HS384`, `HS512`.
```

Comment:
Can `HS*` algorithms listed in 1 line ?

Reply:
I'll unify the lines. However. The Quarkus tooling has some sort of line length limit. If my unifying cause the line to exceed that limit, I'll receive a build error. In response, I must run a script that automatically adds Javadoc line breaks at the required intervals.

Reply:
Updated. Thanks.